Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -141,5 +155,82 @@ private void OnDestroy()
                 Destroy(debugSphere.gameObject);
             }
         }
+
+        private static bool IsObjectPartOfGrabbable(GameObject targetObject, BaseNearInteractionGrabbable grabbable)
+        {
+            return targetObject != null && grabbable != null &&
+                (targetObject == grabbable.gameObject ||
+                // Descendant game objects are grabbable as well.
+                (targetObject.transform != null && grabbable.gameObject.transform != null &&
```

Comment:
Can GameObject.transform ever become null? We don't expect this to happen anywhere else at least. Removing those would make this check a little less verbose.

Reply:
I grabbed this from PokePointer - IMO its redundant. I can clean it up.

Reply:
I removed `gameObject` since `grabbable.transform` is the same thing. Depending on garbage collection, a unity object can be null, but still exist so the component could still exist as well.